app/models/design_file.rb

```diff
@@ -0,0 +1,6 @@
+class DesignFile < ApplicationRecord
+  belongs_to :team
+
+  validates :file_name, presence: true
+  validates :team_id, presence: true
+end
```

Comment:
The 'team_id' field is validated for presence, but there is no validation for its integrity, such as ensuring it references a valid 'team' record. Consider adding a validation to check that the 'team_id' corresponds to an existing team.